### PR TITLE
feat: add more socialite connectors

### PR DESCRIPTION
**Pull request by asbiin**
image

### CODE REVIEW

File: app/Actions/Jetstream/UserProfile.php [hunk 1]
```diff
@@ -16,12 +16,11 @@ class UserProfile
      */
     public function __invoke(Request $request, array $data): array
     {
-        $providers = collect(config('auth.login_providers'))->filter(fn ($provider) => ! empty($provider));
-        $providersName = $providers->mapWithKeys(function ($provider) {
-            return [
-                $provider => config("services.$provider.name") ?? __("auth.login_provider_{$provider}"),
-            ];
-        });
+        $providers = collect(config('auth.login_providers'))->filter(fn ($provider) => ! empty($provider))
```
Comment (djaiss) [inline app/Actions/Jetstream/UserProfile.php]: This is really hard to read, in my opinion. So it's a collection that we filter, and if the provider is not empty, we map each one of these into an array that contains two ternary operators that do something at the end.... Can you make this simpler to read?

File: config/services.php [hunk 2]
```diff
@@ -39,10 +39,50 @@
...
+        'client_id' => env('AZURE_CLIENT_ID'),
+        'client_secret' => env('AZURE_CLIENT_SECRET'),
+        'redirect' => env('AZURE_REDIRECT_URI', '/auth/azure/callback'),
+    ],
+
+    'facebook' => [
+        'client_id' => env('FACEBOOK_CLIENT_ID'),
+        'client_secret' => env('FACEBOOK_CLIENT_SECRET'),
+        'redirect' => env('FACEBOOK_REDIRECT_URI', '/auth/facebook/callback'),
+        'logo' => '/img/auth/facebook.png',
```
Comment (djaiss) [inline config/services.php:52]: The others don't have a logo?

File: resources/js/Pages/Profile/Partials/UpdateProviders.vue [hunk 3]
```diff
@@ -63,29 +62,29 @@ defineProps({
       </div>
 
       <div class="mt-5 space-y-6">
-        <div v-for="provider in providers" :key="provider" class="flex items-center">
-          <img :src="`/img/auth/${provider}.svg`" :alt="provider" class="auth-provider relative" />
+        <div v-for="(provider, id) in providers" :key="id" class="flex items-center">
+          <img :src="provider.logo ?? `/img/auth/${id}.svg`" :alt="provider.name" class="auth-provider relative" />
```
Comment (djaiss) [inline resources/js/Pages/Profile/Partials/UpdateProviders.vue]: Shouldn't this come from the backend (through the view helpers)?

**Review by djaiss — COMMENTED** [review]
A few comments!
Good job!

File: config/services.php (hunk 2 above)
Comment (asbiin) [reply config/services.php:52]: They default to `img/auth/{provider}.svg`

File: resources/js/Pages/Profile/Partials/UpdateProviders.vue (hunk 3 above)
Comment (asbiin) [reply resources/js/Pages/Profile/Partials/UpdateProviders.vue]: ok, good idea

**Review by djaiss — APPROVED** [review]
This is great

**Pull request merged**